Migrations/TabloidDbContextModelSnapshot.cs

```diff
@@ -116,7 +116,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         new
                         {
                             Id = "c3aaeb97-d2ba-4a53-a521-4eea61e59b35",
-                            ConcurrencyStamp = "c0aa97d5-116b-4492-9fb7-753fddbefa21",
```

Comment:
This looks like there may have been similar issues to what we dealt with yesterday. Unsure of why it does this, but could cause merge conflicts? Ignore if it won't

Reply:
I added it to gitignore, hopefully it's not going to updated/changed everytime when we do git push